Resources/Prototypes/Entities/Clothing/Head/hats.yml

```diff
@@ -1134,4 +1134,16 @@
     tags:
     - ClothMade
     - HamsterWearable
-    - WhitelistChameleon
\ No newline at end of file
+    - WhitelistChameleon
+
+#fbi addition
+- type: entity
+  parent: [ClothingHeadBase]
+  id: ClothingHeadHatFBI
+  name: FBI cap
+  description: A cap that look cool to wear, remind you of certain someone.
```

Comment:
this should be changed to "A cap that looks cool, reminds you of a certain someone."

Reply:
```suggestion
  description: A cap that looks cool, reminds you of a certain someone.
```